.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: objective-c
```

Comment:
for swift I think we can use `generic` instead of `objective-c`, maybe it does not matter thought